enigmatry-entry-blueprint-app/src/app/app.module.ts

```diff
@@ -13,29 +13,27 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    HttpClientModule,
-    BrowserAnimationsModule,
-    CoreModule,
-    ApiModule,
-    SharedModule,
-    HomeModule,
-    EntryCommonModule.forRoot(),
-    EntryComponentsModule.forRoot(),
-    AppRoutingModule
-  ],
-  providers: [
-    {
-      provide: APP_INITIALIZER,
-      useFactory: initFactory,
-      deps: [AppInitService],
-      multi: true
-    }
-  ],
-  bootstrap: [AppComponent]
+    declarations: [
+        AppComponent
+    ],
+    bootstrap: [AppComponent],
+    imports: [BrowserModule,
+        BrowserAnimationsModule,
+        CoreModule,
+        ApiModule,
+        SharedModule,
+        HomeModule,
+        EntryCommonModule.forRoot(),
+        EntryComponentsModule.forRoot(),
+        AppRoutingModule],
+    providers: [
+        {
+            provide: APP_INITIALIZER,
+            useFactory: initFactory,
+            deps: [AppInitService],
+            multi: true
+        },
+        provideHttpClient(withInterceptorsFromDi(), withInterceptors([acceptLanguageInterceptor]))
```

Comment:
Mixing both old and new types of registration of interceptors. withInterceptorsFromDi() can be removed when all interceptors are moved to use the new syntax (but this includes adjusting the tests)